src/Http/Presenters/User.php

```diff
@@ -43,7 +43,7 @@ public function table($model)
         return $this->table->of('orchestra.users', function (TableGrid $table) use ($model) {
             // attach Model and set pagination option to true
             $table->with($model);
-            $table->sortable();
+            $table->sortable(['only' => 'fullname', 'email']);
```

Comment:
It should also be possible to filter other fields such as created_at, updated_at and any other custom fields if we want to. I believe using a method would be much easier e.g `getSortableFields()`